src/steamship/plugin/service.py

```diff
@@ -143,7 +125,7 @@ def train(
 
     @abstractmethod
     def train_status(
-        self, request: PluginRequest[TrainStatusPluginInput], model: TrainableModel
+        self, request: PluginRequest[TrainPluginInput], model: TrainableModel
```

Comment:
There is no "distinct" TrainStatusPluginInput anymore -- instead `status.remote_status_input` contains the data to rehydrate the training process.